README.md

```diff
@@ -1,11 +1,11 @@
 # MongooseIM platform
 
-[![Build Status](https://github.com/esl/MongooseIM/workflows/CI/badge.svg)](https://github.com/esl/MongooseIM/actions)
-[![CircleCI](https://circleci.com/gh/esl/MongooseIM.svg?style=shield&circle-token=53cdda11cfb95bd7a7044c509c89cb73d3fdeb5a)](https://github.com/esl/MongooseIM/)
-[![codecov](https://codecov.io/gh/esl/MongooseIM/branch/master/graph/badge.svg)](https://codecov.io/gh/esl/MongooseIM)
+[![GitHub Actions](https://github.com/esl/MongooseIM/workflows/CI/badge.svg?branch=master)](https://github.com/esl/MongooseIM/actions?query=workflow%3ACI+branch%3Amaster)
+[![CircleCI](https://circleci.com/gh/esl/MongooseIM.svg?style=shield&circle-token=53cdda11cfb95bd7a7044c509c89cb73d3fdeb5a)](https://app.circleci.com/pipelines/github/esl/MongooseIM)
```

Comment:
I'm actually not sure the `https://app.circleci.com/pipelines/github/esl/MongooseIM` link is public, I thought it's only visible if you log in to a github account that's under the esl organization. Might be worth checking so, just to be sure.

Reply:
"every ever"? I think the previous link was maybe better as it was public?

Reply:
> "every ever"? I think the previous link was maybe better as it was public?

If you check commit verification statuses they all have the same links to CircleCI, the same story with our PRs.
so yes, these links are everywhere in our repo.

Reply:
They are in the PRs only, in github, not in the source code.